VERSION

```diff
@@ -1 +1 @@
-1.12.1+keybow.2
+2.0.0-rc.0+keybow.1
```

Comment:
:question: (no action needed)

I'm not used to custom patch sets like what we have. Is it best practice to roll the previous patches into a single one when pulling upstream changes?

Reply:
I'm not sure I understood the question. This PR essentially applied the existing Keybow patches to the upstream RPI0 system, although in practice it's physically the other way around since this repo is a fork of the upstream.

If you're talking about the versioning scheme, I followed what was in place from @doughsay, which looks like `<upstream>+keybow.<patch>`, with a patch number that resets to 1 each time the upstream is bumped. For example, in the release history this system went from `v1.11.0+keybow.2` to `v1.11.1+keybow.1`.

Reply:
That second one was the answer, thank you! The patch number that resets each time upstream is bumped is exactly what I wondered.

Reply:
yep, thanks for maintaining the pattern!

Reply:
This should be 1100% semver compatible, which is what I was shooting for